src/vunnel/providers/mariner/__init__.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import os
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from vunnel import provider, result, schema
+from vunnel.providers.mariner.parser import Parser
+
+if TYPE_CHECKING:
+    import datetime
+
+
+@dataclass
+class Config:
+    runtime: provider.RuntimeConfig = field(
+        default_factory=lambda: provider.RuntimeConfig(
+            result_store=result.StoreStrategy.SQLITE,
+            existing_results=provider.ResultStatePolicy.KEEP,
+        ),
+    )
+    request_timeout: int = 125
+    allow_versions: list[str] = field(default_factory=lambda: ["2.0"])
```

Comment:
should we additionally be processing 1.0 versions as well? though it's being sunset soon, it may still be used by users, thus should be supported on grype scans.

Reply:
also, this acs more like a allow list instead of a deny list. Ideally we should be able to accept future versions without needing to update code (similar example here https://github.com/anchore/vunnel/blob/bbe25d971eaf537864846494cfe7cce475ca0e00/src/vunnel/providers/rhel/__init__.py#L26). 

Would that be an ok enhancement to this PR?

Reply:
actually, I take back some of what I said... we do have inconsistencies here between providers: https://github.com/anchore/vunnel/blob/bbe25d971eaf537864846494cfe7cce475ca0e00/src/vunnel/providers/sles/__init__.py#L24

The SLES provider is acting as an allow list as well.

Reply:
We should definitely allow 1.0 to begin with. It's an allow-list because I haven't come up with a great way to glob on a remove git repo, but agree that a (initially empty) denylist would be better.

Reply:
I've added 1.0. Should we add an additional labeled image to the quality tests for 1.0?

Reply:
> Should we add an additional labeled image to the quality tests for 1.0?

if there is a difference in how it's processed, yes. Otherwise there isn't a need to add it. (it looks like it's the same, so no need to add labels for it).